packages/renderer/package.json

```diff
@@ -18,35 +18,35 @@
   "author": "",
   "license": "MIT",
   "devDependencies": {
-    "@types/jest": "29.2.3",
-    "@types/node": "18.11.9",
-    "jest": "29.3.1",
-    "typescript": "4.9.3"
+    "@types/jest": "^26.0.15",
+    "@types/node": "^14.14.6",
+    "jest": "^26.6.3",
+    "typescript": "^4.0.5"
   },
   "dependencies": {
-    "@joplin/fork-htmlparser2": "^4.1.41",
-    "@joplin/fork-uslug": "^1.0.6",
-    "font-awesome-filetypes": "2.1.0",
-    "fs-extra": "10.1.0",
-    "highlight.js": "11.6.0",
-    "html-entities": "1.4.0",
-    "json-stringify-safe": "5.0.1",
-    "katex": "0.13.24",
-    "markdown-it": "13.0.1",
-    "markdown-it-abbr": "1.0.4",
-    "markdown-it-anchor": "5.3.0",
-    "markdown-it-deflist": "2.1.0",
-    "markdown-it-emoji": "2.0.2",
-    "markdown-it-expand-tabs": "1.0.13",
-    "markdown-it-footnote": "3.0.3",
-    "markdown-it-ins": "3.0.1",
-    "markdown-it-mark": "3.0.1",
-    "markdown-it-multimd-table": "4.2.0",
-    "markdown-it-sub": "1.0.0",
-    "markdown-it-sup": "1.0.0",
-    "markdown-it-toc-done-right": "4.2.0",
-    "md5": "2.3.0",
-    "mermaid": "9.1.7"
+    "@joplin/fork-htmlparser2": "^4.1.39",
+    "@joplin/fork-uslug": "^1.0.4",
+    "font-awesome-filetypes": "^2.1.0",
+    "fs-extra": "^8.1.0",
+    "highlight.js": "11.1.0",
+    "html-entities": "^1.2.1",
+    "json-stringify-safe": "^5.0.1",
+    "katex": "^0.13.3",
+    "markdown-it": "^10.0.0",
+    "markdown-it-abbr": "^1.0.4",
+    "markdown-it-anchor": "^5.2.5",
+    "markdown-it-deflist": "^2.0.3",
+    "markdown-it-emoji": "^1.4.0",
+    "markdown-it-expand-tabs": "^1.0.13",
+    "markdown-it-footnote": "^3.0.2",
+    "markdown-it-ins": "^3.0.0",
+    "markdown-it-mark": "^3.0.0",
+    "markdown-it-multimd-table": "^4.0.1",
+    "markdown-it-sub": "^1.0.0",
+    "markdown-it-sup": "^1.0.0",
+    "markdown-it-toc-done-right": "^4.1.0",
+    "md5": "^2.2.1",
+    "mermaid": "^8.13.5"
```

Comment:
Please double-check the versions here! It looks like this PR is downgrading some dependencies.